server/src/controllers/projects/tasks/getTasks.ts

```diff
@@ -0,0 +1,15 @@
+import { Response } from 'express';
+import getTasksQuery from '../../../database/ quieres';
+
+const getTasks = async (req: any, res: Response) => { 
+  const { id } = req.userInformation;
+  try {
+    const { rows, rowCount } = await getTasksQuery(id);
+    if (!rowCount) return res.status(204).json({ msg: 'No Task Yet ' });
```

Comment:
please be consistent, we send msg or message in json body?

Reply:
Done